src/main/java/org/session13/homework/librarymanagementsystem/librarymanagement/Library.java

```diff
@@ -0,0 +1,83 @@
+package org.session13.homework.librarymanagementsystem.librarymanagement;
+
+import org.session13.homework.librarymanagementsystem.profiles.Member;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Library {
+
+    private String libraryName;
+    private List<Member> members;
+    private Map<Book, Integer> availableBooks;
+
+    private Library(String libraryName) {
+        this.libraryName = libraryName;
+        members = new ArrayList<>();
+        availableBooks = new HashMap<>();
+    }
+
+    public static Library createInstance(String libraryName) {
+        return new Library(libraryName);
+    }
+
+    public String getLibraryName() {
+        return libraryName;
+    }
+
+    public List<Member> getMembers() {
+        return members;
+    }
+
+    public void addMember(Member member) {
+        if (member == null) return;
+        if (!members.contains(member)) members.add(member);
+    }
+
+    public void removeMember(Member member) {
+        if (member == null) return;
+        if (members.contains(member)) members.remove(member);
+    }
+
+    public void displayMembers() {
+        members.forEach(System.out::println);
+    }
+
+    public void addBooksToLibrary(Book book, int numberOfCopies) {
```

Comment:
It can be usefull to add JavaDoc comments to your classes and methods to explain what they do, what the parameters are, and what the return values are.